plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -793,63 +778,43 @@ static void receive_video(void *param, struct video_data *frame)
 
 	packet = av_packet_alloc();
 
-#if LIBAVFORMAT_VERSION_MAJOR < 58
-	if (data->output->flags & AVFMT_RAWPICTURE) {
-		packet->flags |= AV_PKT_FLAG_KEY;
-		packet->stream_index = data->video->index;
-		packet->data = data->vframe->data[0];
-		packet->size = sizeof(AVPicture);
+	data->vframe->pts = data->total_frames;
+	ret = avcodec_send_frame(context, data->vframe);
+	if (ret == 0)
+		ret = avcodec_receive_packet(context, packet);
+
+	got_packet = (ret == 0);
+
+	if (ret == AVERROR_EOF || ret == AVERROR(EAGAIN))
+		ret = 0;
+
+	if (ret < 0) {
+		blog(LOG_WARNING,
+		     "receive_video: Error encoding "
+		     "video: %s",
```

Comment:
Can probably unbreak this line.

Reply:
Probably, but that line is not touched in this change and would be an unrelated formatting change.

Reply:
It's moved and the indentation level is reduced, so it's technically touched, but it's not a blocking issue, and it can be addressed some other time. I just dislike broken strings, and broken printed strings are discouraged in the style guide.

Reply:
Turns out I had "hide whitepsace" turned on so it didn't show up on GitHub. Technically the string didn't need to be broken before either since it's not above the column limit so I don't know why that was done.

Reply:
Sure, that's fine.